orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/MonitoredDeployBaseTask.java

```diff
@@ -212,7 +220,7 @@ private TaskResult handleError(
             "Failed to get a valid response from deployment monitor %s, proceeding anyway because this deployment monitor is configured to not failOnError",
             monitorDefinition.getName());
 
-    return buildTaskResult(TaskResult.builder(ExecutionStatus.SUCCEEDED), userMessage);
+    return buildTaskResult(TaskResult.builder(ExecutionStatus.FAILED_CONTINUE), userMessage);
```

Comment:
Good catch :)

Reply:
yeah, i think i was working around the fact that you can't return FAILED_CONTINUE and expect the child stages to resume... now it's possible because of my earlier change #3252